packages/usdk/lib/run-agent.mjs

```diff
@@ -0,0 +1,62 @@
+import { ReactAgentsNodeRuntime } from '../packages/upstreet-agent/packages/react-agents-node/node-runtime.mjs';
```

Comment:
Can we abstract this out a bit?

Seems like this is common functionality with chat.